app/controllers/AuthenticationController.scala

```diff
@@ -639,8 +639,12 @@ class AuthenticationController @Inject()(
       )
   }
 
-  private def acceptTermsOfServiceForUser(user: User, termsOfServiceVersion: Int)(implicit m: MessagesProvider) =
-    organizationService.acceptTermsOfService(user._organization, termsOfServiceVersion)(DBAccessContext(Some(user)), m)
+  private def acceptTermsOfServiceForUser(user: User, termsOfServiceVersion: Option[Int])(
+      implicit m: MessagesProvider): Fox[Unit] =
+    for {
+      acceptedVersion <- Fox.option2Fox(termsOfServiceVersion)
```

Comment:
While being nitpicky 🙈
Could you please add an error message like

```suggestion
      acceptedVersion <- Fox.option2Fox(termsOfServiceVersion) ?~> "Terms of service must be accepted."
```

Reply:
Else some kind of generic 400 error would be returned where an explanation would be useful IMO

Reply:
I can do this of course, no problem. Note that the user would not see 400 anyway because the frontend does not allow not accepting TOS.